docs/auth.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+kcp implements the same RBAC-based authorization scheme as Kubernetes.
+Other authorization schemes (i.e. ABAC) are currently not supported.
+Generally, the same role and role binding priniciples apply the same way as in Kubernetes.
+
+However RBAC rules are always declared within the context of a concrete workspace.
+Rules created in a workspace apply to requests against this workspace independently from the workspace type.
+This is what resemembles Kubernetes authorization model.
+In addition, rules can play a role in authorization to requests to other workspaces. The details will be described below.
+This is true for all non-system workspace types, including root, organizational and enduser.
+The exceptions are system namespaces. These can only be accessed by shard-local admins.
+
+# Authorizers
+
+## Workspace content authorizer
+
+As described above, RBAC rules apply for resources within workspaces.
+They cannot express the basic authorization primitive whether a subject is generally
+authorized to access a workspace.
+
+The lack of any bindings is not sufficient to express general access to a workspace
+as every authenticated user in Kubernetes is at least bound to `system:authenticated`
+with bindings against at least some cluster roles, like `system:basic-user`, `system:discovery`,
+and `system:public-info-viewer`.
+
+Expressing basic access to a workspace via bindings against the `ClusterWorkspace` resource type
+is not sufficient either because the permission to mutate the `ClusterWorkspace` resource
+(i.e. changing its metadata) is orthogonal to accessing its contained resources.
+
+Hence, an additional authorization layer is necessary. To express authorization patterns,
+kcp recognizes role bindings defined against the `clusterworkspaces/content` resource under the `tenancy.kcp.dev` api group.
+This subresource is not backed by any CRD but is known to the workspace content authorizer.
```

Comment:
this is too much about motiviation. Can we be very clear how the system behaves rather than saying what wouldn't be enough?